packages/Popover/src/components/Content/Content.js

```diff
@@ -64,6 +65,10 @@ const Content = React.forwardRef((props, ref) => {
     }, parseInt(PopoverConstants.transition, 10));
   }
 
+  function cssValue(value) {
```

Comment:
`convertToPx` instead of `cssValue`? A bit more descriptive

Reply:
Or i guess you are handling other values such as `100%` in which case i guess you won't want the conversion...

Reply:
Yeah, it's only adding `px` to numbers, if it's something like `100%` or `auto`, then I leave it as is.